context/ethereumContext.tsx

```diff
@@ -453,7 +453,11 @@ export const EthereumProvider: React.FC<{}> = ({ children }) => {
     const meta = OpcodesMeta as IReferenceItemMetaList
     // TODO: need to implement proper selection of doc according to selected fork (maybe similar to dynamic gas fee)
     // Hack for "difficulty" -> "prevrandao" replacement for "merge" HF
-    if (selectedFork?.name === mergeHardforkName && toHex(op.code) == '44') {
+    if (
+      selectedFork?.name != null &&
```

Comment:
nit: same comment as above (function for the fork check)

Reply:
makes sense 👍